src/main/jte/layout.jte

```diff
@@ -82,19 +82,29 @@
                     </div>
                     <div class="navbar-end">
                         @if (layout.getCurrentHighestBooking() != null)
-                            <div class="badge badge-neutral m-3">p-${context.localize("bookings.priority." + layout.getCurrentHighestBooking().getPriority())}</div>
+                            <div class="badge badge-neutral m-3" aria-label="${context.localize("bookings.priority")}" title="${context.localize("bookings.priority.tooltip")}">
+                                p-${context.localize("bookings.priority." + layout.getCurrentHighestBooking().getPriority())}</div>
                         @endif
 
                         !{var currentHighestBooking = layout.getCurrentHighestBooking();}
                         @if (currentHighestBooking != null)
                             !{var bookingShareType = currentHighestBooking.getShareRoomType();}
                             @if(bookingShareType == ShareRoomType.YES)
-                                <i class="fa-solid fa-users"></i>
+                                <label class="label" aria-label="${context.localize("bookings.collaboration.share_room")}"
```

Comment:
Don't repeat it for `aria-label` and `title`.
Also, I think just adding the label to the `i` element would be nicer